common/src/main/scala/hmda/model/filing/ts/TsFieldMapping.scala

```diff
@@ -17,6 +17,6 @@ object TsFieldMapping {
     "Total Number of Entries Contained in Submission" -> ts.totalLines.toString,
     "Federal Taxpayer Identification Number" -> ts.taxId,
     "Legal Entity Identifier (LEI)" -> ts.LEI,
-    "All data fields in the LAR" -> ts.toCSV
+    "All data fields in the LAR" -> "Error - mismatch"
```

Comment:
I believe that we want this one to say: `Your LAR contains one or more duplicate records.`